tests/testthat/test-pkgdown.R

```diff
@@ -0,0 +1,29 @@
+test_that("can parse pkgdown lists", {
```

Comment:
Should this file be called test-pkgdepends.R ?

Reply:
yes